update: exceptions to 50 users limit per service (#867)
aiven/aiven-docs: docs/platform/howto/create_new_service_user.md

and Aiven for Grafana®.

:::note
By default, the maximum amount of users allowed on a service is 50. This limitation doesn't

> apply to the following services:

Reviewer: services -> Aiven services? more consistent with the wording above about Flink

Reviewer: I mean: "This limitation does not apply to the following Aiven services"

Author: yeah, to be perfectly precise "Aiven-managed services"

> By default, the maximum amount of users allowed on a service is 50. This limitation doesn't
> apply to the following Aiven-managed services:

Reviewer 2: ```suggestion
By default, Aiven-managed services allow up to 50 users.
This limit does not apply to the following services:
```
Small wording suggestion
